components/grid.html

```diff
@@ -0,0 +1,190 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Gimesto</title>
+    <link rel="icon" type="image/png" sizes="32x32" href="../assets/favicon/favicon-32x32.png">
+    <link rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/styles/atom-one-dark.min.css">
+    <link rel="stylesheet" href="../gimesto.css" type="text/css" />
+  </head>
+  <body>
+    <!-- Header Section Start -->
+    <header class="header">
+        <div class="logo">
+            <a href="../index.html">Gimesto</a>  
+        </div>
+            <ul class="header-nav">
+                <li>
+                    <a href="https://github.com/vrx29/">
+                        <i class="bi bi-github github-icon"></i>
+                         github
+                    </a>
+                </li>
+                <li>
+                    <button class="menu">
+                        <i id="menu-icon" class="bi bi-list"></i>
+                    </button>
+                </li>
+            </ul>  
+    </header>
+    <!-- Header Section End -->
+
+    <!-- Main Section Start -->
+    <main class="doc-container">
+        <!-- Sidebar Section Start -->
+        <nav class="sidebar">
+            <!-- Getting Started Navigation Start -->
+            <div>
+                <div class="list-topic">
+                    <a  href="#">GETTING STARTED</a>
+                </div>
+                <ul class="list-subtopic">
+                    <li >
+                        <a href="/components/installation.html">Installation</a>
+                    </li>
+                    <li>
+                        <a href="/components/colors.html">Colors</a>
+                    </li>
+                    <li>
+                        <a href="/components/typography.html">Typography</a>
+                    </li>
+                </ul>
+            </div>
+            <!-- Getting Started Navigation End -->
+
+            <!-- Components Navigation Start -->
+           <div>
+                <div class="list-topic">
+                    <a class="" href="#">COMPONENTS</a>
```

Comment:
Use `javascript:;` instead of `#` in href

Reply:
done